### PR TITLE
Add statoscope

### PR DESCRIPTION
**Pull request by smelukov**
It's all about #217

### CODE REVIEW

File: packages/size-limit/create-help.js [hunk 1]
```diff
@@ -43,6 +43,9 @@ module.exports = process => {
         '',
         b('Webpack options:'),
         `  ${y('--why')}              Show package content`,
+        `  ${y(
+          '--other-stats'
```
Comment (ai) [inline packages/size-limit/create-help.js:47]: `--compare-with`

File: yarn.lock [hunk 2]
```diff
@@ -9,186 +9,186 @@
...
-  version "7.15.0"
-  resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.15.0.tgz#2dbaf8b85334796cafbb0f5793a90a2fc010b176"
-  integrity sha512-0NqAC1IJE0S0+lL1SWFMxMkz1pKCNCjI4tr2Zx4LJSXxCLAdr6KyArnY+sno5m3yH9g737ygOyPABDsnXkpxiA==
+"@babel/compat-data@^7.13.8":
+  version "7.13.8"
+  resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.13.8.tgz#5b783b9808f15cef71547f1b691f34f8ff6003a6"
+  integrity sha512-EaI33z19T4qN3xLXsGf48M2cDqa6ei9tPZlfLdb2HC+e/cFtREiRd8hdSqDbwdLB0/+gLwqJmCYASH0z2bUdog==
 
 "@babel/core@^7.1.0", "@babel/core@^7.12.16", "@babel/core@^7.7.5":
-  version "7.15.5"
```
Comment (ludofischer) [inline yarn.lock:25]: Why do the `yarn.lock` changes downgrade the peer dependencies?

Comment (ai) [reply yarn.lock:25]: I will call `yarn upgrade` after the merge anyway

**Comment by ai** [comment]
Can you try to check that `npx size-limit --why` works?

I tried on [`nanostores/persistent`](https://github.com/nanostores/persistent) and got 404 in the browser, because Size Limit removed temporary file.

**Comment by smelukov** [comment]
I'm without my laptop now, but if size-limit removes build directory then we can use os tmp directory to save statoscope report 🤔

**Pull request merged**